.eslintrc.json

```diff
@@ -0,0 +1,39 @@
+{
+  "env": {
+    "browser": true,
+    "es2020": true,
+    "mocha": true
+  },
+  "extends": [
+    "airbnb-base",
+    "plugin:import/typescript"
```

Comment:
Overall LGTM.  I noticed one lint for example:

```
fullstory-data-layer-observer/test/utils/mocha.ts
   3:10  error  'MockClass' is defined but never used                                  no-unused-vars
   3:21  error  'Call' is defined but never used                                       no-unused-vars
```

That seems odd since the import is only providing type safety.  Given this [discussion on SO](https://stackoverflow.com/questions/57802057/eslint-configuring-no-unused-vars-for-typescript), looks like there's a recommended rule set.  But rather than take it en massed, it can be scoped down to a specific rule:

```
{
      "files": ["*.ts"],
      "rules": {
        "no-unused-vars": "off",
        "@typescript-eslint/no-unused-vars": ["error"]
      }
    }
```

I'll include a linted PR with the above and fixes.